app/controllers/api/v2/claims/build_claims_controller.rb

```diff
@@ -15,7 +15,8 @@ def create
             # This is a bit of a frig - because we are not expecting the caller to
             # add the AssignReferenceToClaim command, we cant expect them to check the meta for
             # that command so we pretend its from the BuildClaim command instead
-            result.meta['BuildClaim'] = result.meta.delete('AssignReferenceToClaim')
+            result.meta['BuildClaim'] ||= {}
```

Comment:
The BuildClaim command now returns its own meta - so this was overwriting it before